architectures/models/SSDNet.py

```diff
@@ -15,16 +15,16 @@ def __init__(self, in_channels, n_classes, k):
         self.k = k
         '''
         idea: adding a DWSC is cheap anyway, so might as well try to use it to 'specialize' features onto this particular output from the main stream
-        '''
+        # '''
```

Comment:
Why are these quotation marks commented?

Reply:
The quo

> Why are these quotation marks commented?

The quotes are not commented, the comment is quoted.